vaadin-grid-flow-parent/vaadin-grid-flow-integration-tests/src/main/java/com/vaadin/flow/component/grid/it/GridLitRendererPage.java

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright 2000-2021 Vaadin Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package com.vaadin.flow.component.grid.it;
+
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+
+import com.vaadin.flow.component.Component;
+import com.vaadin.flow.component.grid.Grid;
+import com.vaadin.flow.component.html.Div;
+import com.vaadin.flow.component.html.NativeButton;
+import com.vaadin.flow.data.renderer.ComponentRenderer;
+import com.vaadin.flow.renderer.LitRenderer;
+import com.vaadin.flow.router.Route;
+
+@Route("vaadin-grid/lit-renderer")
+public class GridLitRendererPage extends Div {
+
+    public GridLitRendererPage() {
+        Grid<Integer> grid = new Grid<>();
+        grid.setItems(
+                IntStream.range(0, 1000).boxed().collect(Collectors.toList()));
+        grid.addColumn(LitRenderer
+                .<Integer> of(
+                        "<span id=\"item-${index}\">Lit: ${item.name}</span>")
+                .withProperty("name", item -> "Item " + item));
```

Comment:
Can I ask for a test, that verifies that using two `LitRenderer` with the same `property` name can be used in two seperat columns without overriding the previous set `property`? :) Xref: https://github.com/vaadin/flow-components/issues/916#issuecomment-832493018

Reply:
👍 Thanks for the suggestion. Will add the test.

Reply:
Covered in https://github.com/vaadin/flow-components/pull/1838/commits/048d4376c7ac21b9db9d5688272dd6399a1d2271